src/inference/include/openvino/runtime/intel_cpu/properties.hpp

```diff
@@ -12,6 +12,7 @@
 #pragma once
 
 #include "openvino/runtime/properties.hpp"
+#include "openvino/runtime/system_conf.hpp"
```

Comment:
you cannot mix development and public API headers.

I see the processor type is currently in dev api, why do we need to re-include it here?

Reply:
Because `using ov::ProcessorType;` in this header file. Any suggestion for using it?

Reply:
Updated. Move processor type definition out of dev_api.